src/airflow/dags/airnow_bam_data.py

```diff
@@ -84,6 +84,7 @@ def send_to_message_broker(data: pd.DataFrame):
         data = DataValidationUtils.process_for_message_broker(
             data=data, tenant=Tenant.US_EMBASSY
         )
+        print(data)
```

Comment:
@NoahNsimbe I only log the measurements from here right?

Reply:
No. don't log here, log before sending to the message broker. @BenjaminSsempala

Reply:
Thanks @NoahNsimbe I've made the changes